Header header
geometry_msgs/Pose pose
float64 velocity
int16 heading

